tests/unit/base_tests_zarrio.py

```diff
@@ -988,12 +983,7 @@ class BaseTestExportZarrToZarr(BaseZarrWriterTestCase):
     """
 
     def setUp(self):
-        self.store = [
-            get_temp_filepath(),
-            get_temp_filepath(),
-            get_temp_filepath(),
-            get_temp_filepath()
-        ]
+        self.store = [f"file{i}.zarr" for i in range(3)]
```

Comment:
What is the difference between `self.store` and `self.store_path`? Why have two different variables that are the same?